Canvas/CanvasUITests/Quizzes/QuizTests.swift

```diff
@@ -42,4 +42,32 @@ class QuizTests: CanvasUITests {
         Quiz.text(string: "This is question D").waitToExist()
         XCTAssertEqual(XCUIElementWrapper(app.textFields.firstMatch).value, "42.000000")
     }
+
+    func testQuizQuestionsOpenInWebView() {
+        Dashboard.courseCard(id: "263").waitToExist()
+        Dashboard.courseCard(id: "263").tap()
+        CourseNavigation.quizzes.tap()
+
+        app.find(labelContaining: "Web Quiz").tap()
+        Quiz.resumeButton.tapAt(.zero)
+        Quiz.resumeButton.tapAt(.zero)
+
+        Quiz.text(string: "Question 1").waitToExist()
+        Quiz.text(string: "Question 2").waitToExist()
+        let textFields = app.textFields.allElementsBoundByIndex
+        for textField in textFields {
+            if let value = textField.value as? String {
+                XCTAssert(value == "Fox" || value == "Dog" || value == "6.4")
+            } else {
+                XCTAssert(false, "text field did not have a value")
+            }
+        }
+    }
+
+    func testQuizzesShowEmptyState() {
+        Dashboard.courseCard(id: "262").waitToExist()
+        Dashboard.courseCard(id: "262").tap()
+        CourseNavigation.announcements.waitToExist()
+        CourseNavigation.quizzes.waitToVanish()
```

Comment:
Im not sure this is what we want. This test would be tapping into Quizzes and verifying that the list is empty.. Should look for this label:

<img width="272" alt="image" src="https://user-images.githubusercontent.com/1187499/61490951-4c682800-a96b-11e9-8838-9d2784ac041b.png">

Reply:
If a course has no quizzes then that page doesn't appear as an option in the course navigation. So I just checked that the option wasn't there because I don't know any other way to the quizzes page otherwise.

Reply:
Oh gotcha. Makes sense. I had checked in Teacher. Good good.

Reply:
Unless you want me to put link somewhere that brings me to that page?

Reply:
No, no. What you have is fine. I was confused.